docs/guide/2.x/breaking-changes.md

```diff
@@ -0,0 +1,22 @@
+# Breaking changes
```

Comment:
Most likely should be declared at GitHub releases instead.

Reply:
ack.

Reply:
hm, I think it might get lost there though, typically projects have separate doc page covering migration between major releases and it's kinda nice

Reply:
I think change log can go to GH releases and we just link to it from the doc. We should still have a `Migration Guide` page in the docs as well.